tests/base/test_configset.lua

```diff
@@ -177,14 +177,20 @@
 		local f = field.get("flags")
 		configset.store(cset, f, { "Symbols", "WinMain", "MFC" })
 		configset.remove(cset, f, { "WinMain" })
-		test.isequal({ "Symbols", "MFC" }, configset.fetch(cset, f, {}))
+
+		local result = configset.fetch(cset, f, {})
+		test.print(table.tostring(result))
```

Comment:
Yes, deliberately committed.. we very rarely see these two tests fail, and I want to have some extra output when it occurs.